contracts/external/cw-admin-factory/Cargo.toml

```diff
@@ -16,17 +16,16 @@ backtraces = ["cosmwasm-std/backtraces"]
 library = []
 
 [dependencies]
-cosmwasm-std = "1.0.0"
-cosmwasm-storage = "1.0.0"
-cw-storage-plus = "0.13.2"
-cw2 = "0.13.2"
-schemars = "0.8.8"
-serde = { version = "1.0.137", default-features = false, features = ["derive"] }
+cosmwasm-std = "1.1"
+cosmwasm-storage = "1.1"
+cosmwasm-schema = "1.1"
+cw-storage-plus = "0.15"
+cw2 = "0.15"
```

Comment:
could we bump all of our CosmWasm dependencies as part of this change? still some 0.13 in here.

Reply:
Yes please.

Reply:
introduced the workspace-inheritance!